fuzzers/ensemble/fuzzer.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Integration code for AFL fuzzer."""
+
+import collections
+import shutil
+import subprocess
+import os
+import tempfile
+
+from fuzzers import utils
+
+# OUT environment variable is the location of build directory (default is /out).
+
+# Fuzz each fuzzer for 1 hour before rotating.
+SECONDS_PER_FUZZER = 3600
+
+
+def prepare_build_environment():
+    """Set environment variables used to build AFL-based fuzzers."""
+    utils.set_no_sanitizer_compilation_flags()
+
+    cflags = ['-O3', '-fsanitize-coverage=trace-pc-guard']
+    utils.append_flags('CFLAGS', cflags)
+    utils.append_flags('CXXFLAGS', cflags)
+
+    os.environ['CC'] = 'clang'
+    os.environ['CXX'] = 'clang++'
+    os.environ['FUZZER_LIB'] = '/libAFL.a'
+
+
+def build():
+    """Build benchmark with AFL."""
+    prepare_build_environment()
+
+    utils.build_benchmark()
+
+    print('[post_build] Copying afl-fuzz to $OUT directory')
+    # Copy out the afl-fuzz binary as a build artifact.
+    shutil.copy('/afl/afl-fuzz', os.path.join(os.environ['OUT'], 'afl'))
+    shutil.copy('/aflpp/afl-fuzz', os.path.join(os.environ['OUT'], 'aflpp'))
+    shutil.copy('/aflsmart/afl-fuzz', os.path.join(os.environ['OUT'],
+                                                   'aflsmart'))
+    shutil.copy('/fair/afl-fuzz', os.path.join(os.environ['OUT'], 'fair'))
+    shutil.copy('/mopt/afl-fuzz', os.path.join(os.environ['OUT'], 'mopt'))
+
+
+def prepare_fuzz_environment(input_corpus):
```

Comment:
Does this function and run_afl_fuzz need to be copied from afl/fuzzer.py?
Or can you simply import them?